docker-compose.datadog.yml

```diff
@@ -0,0 +1,20 @@
+version: "2.3"
+services:
+  datadog:
+    image: datadog/agent:latest
```

Comment:
datadog agent is added to send metrics during circle ci test runs.
example of failure - https://app.circleci.com/pipelines/github/omgnetwork/elixir-omg/8939/workflows/bcebdc24-e205-4f32-b009-64455735beb2/jobs/87535 - I added sleep(100) to simulate it

Reply:
FYI: datadog is also defined in `docker-compose.dev.yml` 
maybe they could be merged?